### DISCUSSION TITLE
Parental control (Родительский контроль)

### DISCUSSION THREAD

**Discussion by Absaz** [AdguardTeam/AdGuardHome#4041] [Q&A]
It is necessary to configure filtering rules for the devices used by the child. All of these devices have a "user_child" tag. It must be done so that these devices can only have access to those Internet resources that will be included in the rule, and all other resources are not available by default. How to properly implement this functionality using Custom Filtering Rules?

(Необходимо настроить правила фильтрации для устройств, которыми пользуется ребёнок. Все эти устройства имеют тэг "user_child". Нужно сдалть так, чтобы эти устройства могли иметь доступ только к тем интернет-ресурсам, которые будут внесены в правило, а все остальные ресурсы были недостпуны по-умолчанию. Как правильно реализовать этот функционал с помощью Пользовательских правил фильтрации?)

**Comment by ainar-g** [comment]
Добрый день!

Это можно сделать через раздел «Фильтры → Пользовательские правила фильтрации» с помощью правил с модификатором [`ctag`][1].  Например, чтобы запретить клиентам с тэгом `user_child` всё, кроме домена `ru.wikipedia.org`:

```none
*$ctag=user_child
@@|ru.wikipedia.org^$ctag=user_child
```

[1]: https://github.com/AdguardTeam/AdGuardHome/wiki/Hosts-Blocklists#ctag
